media-gfx/openvdb/openvdb-4.0.2-r3.ebuild

```diff
@@ -57,6 +61,8 @@ PATCHES=(
 	"${WORKDIR}/${P}-patchset-02/0002-use-pkgconfig-for-ilmbase-and-openexr.patch"
 	"${WORKDIR}/${P}-patchset-02/0003-boost-1.65-numpy-support.patch"
 	"${FILESDIR}/${P}-findboost-fix.patch"
+	"${FILESDIR}/${P}-fix-const-correctness-for-unittest.patch"
+	"${FILESDIR}/${P}-fix-build-docs.patch"
```

Comment:
The convention is ${P} for upstreamed patches which should be in the next release btw. Not sure if that's the case here but just sharing in case.

Reply:
These patches are used in several releases before being fixed in either openvdb 6 or 7. If I use ${P} for the version where it is first introduced and then ${PN}-x.x.x for subsequent versions that will avoid requiring multiple copies of the same patch. Is there a better way to do it?

Reply:
I think in this case, there's no need to worry. Include the information in the patch if you can but it was more just a little suggestion if it was feasible. No need to trouble yourself here!